verticaltabs.js

```diff
@@ -843,6 +843,25 @@ VerticalTabs.prototype = {
     }
   },
 
+  delayResizeTabs: function (delay) {
+    if (delay <= 0) {
```

Comment:
nit: This could be `delay < 0`, so that we don't bother re-setting a `delay` of `0` to `0`.  🙂